### PR TITLE
React Challenge: Adds Inline options 

### PR DESCRIPTION
**Pull request by takshch**
# Description
It adds the React challenge of inline options(or segmented items).

# Screenshots
Screenshot from 2024-02-18 22-12-25

# Suggestions needed
Inline Options is quite a general name.
I think we can improve the name of this challenge.
Please suggest a name if you have any in mind.

### CODE REVIEW

**Comment by takshch** [comment]
@sadanandpai Can you please review this PR?

**Review by sadanandpai — APPROVED** [review]

**Comment by sadanandpai** [comment]
plz resolve the conflicts @takshch

**Comment by sadanandpai** [comment]
inline options may not give the ppl an idea about the component. I don't have a better name for now. But if you get one better name do update in the future

File: apps/react/src/challenges/inline-options/InlineOptions.module.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.option-selected {
+  outline: none;
+  color: #fff;
+  background: #6a604f;
+}
+
+.option-disabled {
+  color: #bfbdbd;
+  cursor: default;
```
[the remark this answers is not in the document]
Comment (takshch) [reply apps/react/src/challenges/inline-options/InlineOptions.module.css]: Done!

**Comment by takshch** [comment]
> plz resolve the conflicts @takshch

Done!

**Comment by takshch** [comment]
> inline options may not give the ppl an idea about the component. I don't have a better name for now. But if you get one better name do update in the future

At Print Intuitively, We named it **segmented**.
This website is called "Segmented Buttons".
Please take a look at it: https://m3.material.io/components/segmented-buttons/overview
More: https://medium.com/tap-to-dismiss/a-better-segmented-control-9e662de2ef57

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**